### ISSUE TITLE
implementar posible tutorial de uso

### ISSUE THREAD

**Issue by fabian199827** [ODES-Chile/odes-unidades-shiny-app#66]
Hacer un recorrido de la aplicación como en EO browser:

Agregar opción de info
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/997eeae2-730e-4c23-a084-0fa9c998f4c6)

Cuando se haga clic, que se desplieguen lo siguiente:
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/c676ed93-0065-4f6e-b9c3-2c422c58c6be)
y con una opción a un tutorial de uso

Cuando se le da clic a la opción del tutorial, se da un recorrodo guiado a todas las opciones de la aplicación, creo que esto simplificaría el uso para el ususario
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/1edc13f8-ac8f-4b90-8b47-1bd9dd225357)

**Comment by jbkunst** [comment]
Puiedes ir agregando acá una lista de cada item y descripción, la cual yo podré tomar e implementar, por ejemplo:

1. Selector de macrozona: Con este selector es posible elegir cualquier macrozona, con el fin de visualizar información de cualquier macrozona en específico.
2. Selector de "Unidad administrativa": Esta pestaña permite elegir entre 7 unidades administrativas dististintas: Región, provincia, comuna, distrito censal, cuenca, subcuenca y subsubcuenca. Con el fin de obtener diversa información climática agregada a distintas unidades administrativas 
3. Selector de variable: Aquí es posible seleccionar distintas variables climáticas, entre precipitación, temperatura, humedad relativa, evapotranspiración, humedad de suelo e indicadores de sequía y de vegetación.
4. Selector de fecha: permite seleccionar distintos intervalos de fecha, desde enero de 1981 a mayo de 2023. El intervalo seleccionado se visualiza en el mini gráfico.
5. Mini gráfico: Aquí se visualiza la serie de tiempo de la variable y fechas selecionadas. También es posible descargar los datos brutos en formato Excel.
6. Mapa: En el mapa se muestran las variables seleccionadas en el panel lateral. Aquí se puede seleccionar cualquier polígono y obtener el reporte de sequía agregado a la unidad administrativa seleccionada.
7. Reporte de sequía: Reporte que muestra el estado de sequía en que se encuentra la unidad administrativa seleccionada. Cada indicador de sequía posee un informativo que explica a lo que se refiere cada uno. También es posible descargar los datos brutos de los indicadores presentados en el reporte.

Que les parece? opiniones? @jbkunst @frzambra @joaquinRiquelme

**Comment by jbkunst** [comment]
Ok, en un principio intentaré implementarlo con una funcionalidad similar, en ultimo caso copiamos el texto a la sección ayuda.

@fabian199827

**Comment by fabian199827** [comment]
@jbkunst Estaba viendo lo de la ayuda quedó super bien a mi parecer, pero hay algunos detallitos de forma que me equivoqué yo en el texto que escribí y me salta una pregunta igual que la podríamos conversar hoy:

Detalle de forma y arreglo en la redacción de una parte:
1)  Poner "permite" con mayúscula"
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/fe4ca628-2adb-463a-9d29-7e4500723cfe)
 2) Cambiar lo resaltado por: "También es posible descargar los datos brutos del periodo de tiempo seleccionado en formato Excel".
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/dd151410-443e-48dc-aed4-163eb6c25a99)
3) Borrar la palabra "Reporte" repetida
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/08781f52-58b6-4a84-8e70-5499d56ea576)

Pregunta:
En esta parte:
![imagen](https://github.com/ODES-Chile/odes-unidades-shiny-app/assets/128641014/4efac679-00d4-4723-8e65-bc3c06a38a27)
La fecha irá cambiando mes a mes cada vez que se actualicen los datos, ahora está hasta junio, cómo lo haremos ahí? hay que estarlo cambiando constantemente? o lo dejamos simplemente como: 

1)"Permite seleccionar distintos intervalos de fecha, desde 1981 a 2023. El intervalo seleccionado se visualiza en el mini gráfico." (en este caso igual tendremos que cambiarlo cuando pasemos a 2024)
2) "Permite seleccionar distintos intervalos de fecha. El intervalo seleccionado se visualiza en el mini gráfico".

Ahí lo conversamos...

**Comment by jbkunst** [comment]
Hola @fabian199827 ,

Notar que este texto lo pudes modificar al igual modificas el html de la página de ODES :).

El archivo a modificar es texto plano: https://github.com/ODES-Chile/odes-unidades-shiny-app/blob/main/md/ayuda.md

**Comment by fabian199827** [comment]
Hola @jbkunst una pregunta... hice los cambios en el texto plano pero no se cambia en la plataforma en la parte de "ayuda", esperé pero nada... debo hacer algo extra a parte de darle clic en "commit changes"? o debes hacer algo extra tu?

**Comment by jbkunst** [comment]
sí, hay que hacer algo extra, se debe hacer el deploy, subir todo de nuevo al servicio, el cual incluye tus cambios. En la tarde lo subo!
